ops/c/src/ompoffload/ops_ompoffload_common.cpp

```diff
@@ -130,3 +133,18 @@ void ops_device_memset(OPS_instance *instance, void** ptr, int val, size_t size)
 
 void ops_device_sync(OPS_instance *instance) {
 }
+
+void ops_randomgen_init(unsigned int seed, int options) {
+  ops_randomgen_init_host(seed, options, ops_rand_gen);
```

Comment:
Aignature changed here and in the rest, ops_rand_gen needs to be passed by reference

Reply:
ops_randomgen_init_host function in ops_lib_core accept the argument as pass by reference. No change will be required at the calling side right?

Reply:
Yes, no change on the calling side